tests/elasticache.rs

```diff
@@ -2,7 +2,7 @@
 
 extern crate rusoto;
 
-use rusoto::elasticache::{ElastiCacheClient, DescribeCacheClustersMessage};
+use rusoto::elasticache::{Elasticache, ElastiCacheClient, DescribeCacheClustersMessage};
```

Comment:
The new trait import should be `ElastiCache`.

```bash
error[E0432]: unresolved import `rusoto::elasticache::Elasticache`
 --> tests/elasticache.rs:5:27
  |
5 | use rusoto::elasticache::{Elasticache, ElastiCacheClient, DescribeCacheClustersMessage};
  |                           ^^^^^^^^^^^ no `Elasticache` in `elasticache`. Did you mean to use `ElastiCache`?
```

Taking that compiler suggestion makes it work. 😄